codegen_glibmm/templates/proxy.h.templ

```diff
@@ -83,6 +84,7 @@ public:
     void {{ prop.name }}_set({{ prop.cpptype_in }}, const Gio::SlotAsyncReady &);
     void {{ prop.name }}_set_finish(const Glib::RefPtr<Gio::AsyncResult> &);
     {% endif %}
+    sigc::signal<void> {{ prop.name }}_changed;
```

Comment:
I would prefer if this was a function: this in the future could allow us to refactor the way that signals work, for example avoid instantiating them until someone is actually interested in them.

Reply:
Yes, good idea. Especially if there are as many properties as in the many-types example.